test/spec/modules/relaidoBidAdapter_spec.js

```diff
@@ -11,6 +11,7 @@ const setUAMobile = () => { window.navigator.__defineGetter__('userAgent', funct
 
 describe('RelaidoAdapter', function () {
   const relaido_uuid = 'hogehoge';
+  window.document.cookie = `${UUID_KEY}=${relaido_uuid}`
```

Comment:
please do not use direct cookie setting in your tests.

Unit tests are run across all browsers and this causes flaky results at times.

Instead, you should stub the utils.getCookie and utils.setCookie functions to perform whatever you want.

Reply:
Thank you for the review.

For various reasons, close this pull request

I will send you a new pull request that includes this one immediately.

I'll fix the test code at that time.